src/controller/audio-stream-controller.js

```diff
@@ -159,7 +157,7 @@ class AudioStreamController extends BaseStreamController {
         // if track info not retrieved yet, switch state and wait for track retrieval
         if (typeof trackDetails === 'undefined') {
           this.state = State.WAITING_TRACK;
-          break;
+          return;
```

Comment:
@gkolb Why do we return from the function here instead of breaking the loop?

Reply:
I think I was just trying to be consistent with the stream-controller.js idle case in the doTick method. It shouldn't make a difference because there is no logic outside of the switch statement but I can try to update this.

Reply:
I like the consistency with the stream-controller IDLE case, this is fine with me since there's no logic after the loop